### PR TITLE
Refactor partv

### PR DESCRIPTION
**Pull request by BiliouriV**

### CODE REVIEW

File: Projects/l0/Led_Fader/inc/main.h [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+/* Exported functions prototypes ---------------------------------------------*/
+void Error_Handler(void);
+
+/* USER CODE BEGIN EFP */
+
+/* USER CODE END EFP */
+
+/* Private defines -----------------------------------------------------------*/
+#define B_Pin GPIO_PIN_0
```
Comment (Salem-Luos) [inline Projects/l0/Led_Fader/inc/main.h:72]: moify cube mx with only what you need and regenerate the project

File: Projects/l0/Bike_alarm/Start_controller/lib/Button/library.json [hunk 2]
```diff
@@ -2,21 +2,16 @@
...
-    "build": {
-        "flags": [
-            "-DREV={0,7,0}"
-        ]
-    },
     "dependencies": [
         {
             "name": "luos/Luos",
-            "version": "^1.0.0"
+            "version": "^1.3.0"
```
Comment (Salem-Luos) [inline Projects/l0/Bike_alarm/Start_controller/lib/Button/library.json]: 2.0.0

File: Projects/l0/Potentiometer/lib/Potentiometer/potentiometer.c [hunk 3]
```diff
@@ -123,8 +58,11 @@ static void Potentiometer_MsgHandler(service_t *service, msg_t *msg)
         msg_t pub_msg;
         // fill the message infos
         pub_msg.header.target_mode = ID;
-        pub_msg.header.target      = msg->header.source;
+        // the message destination is the service that asked information
```
Comment (Salem-Luos) [inline Projects/l0/Potentiometer/lib/Potentiometer/potentiometer.c:61]: small improve : send value of potentiommetre only when angle is differente from prev value

**Review by Salem-Luos — CHANGES_REQUESTED** [review]
Nice!!!

**Review by ncros3 — APPROVED** [review]
Good job.

**Pull request merged**